site/content/en/docs/installation/_index.md

```diff
@@ -52,6 +52,13 @@ VERSION={{< param "version" >}}
 kubectl apply --server-side -f https://github.com/kubernetes-sigs/jobset/releases/download/$VERSION/manifests.yaml
 ```
 
+You can also install using a released helm chart as follows
+
+```shell
+helm install jobset oci://us-central1-docker.pkg.dev/k8s-staging-images/charts/jobset --version=$VERSION --create-namespace --namespace=jobset-system
```

Comment:
```suggestion
helm install jobset oci://jobset oci://registry.k8s.io/charts/jobset --version=$VERSION --create-namespace --namespace=jobset-system
```

Registry is not user facing and I'm not sure the TTL for these images.

Reply:
do you know what would the uninstall instruction look like? I can add that as well.

Reply:
You can refer Kueue documentation, I believe: https://kueue.sigs.k8s.io/docs/installation/#uninstall

Reply:
Thanks @tenzen-y, I added that now.

@kannon92 this didn't work:

```
ahg@ahg-cloudtop:~/helm$ ./helm install jobset oci://registry.k8s.io/charts/jobset --version=$VERSION --create-namespace --namespace=jobset-system
Error: INSTALLATION FAILED: Unable to locate any tags in provided repository: oci://registry.k8s.io/charts/jobset
```

Reply:
```
kehannon@kehannon-thinkpadp1gen4i:~/Work/openshift/kueue-operator$ helm install jobset oci://registry.k8s.io/charts/jobset --version=v0.8.0 --create-namespace --namespace jobset-system
Pulled: registry.k8s.io/charts/jobset:v0.8.0
Digest: sha256:e1f25f53ba4312f99bb60b92668d98104c549067d86756d30cbf1cf18f2b47f7
NAME: jobset
LAST DEPLOYED: Mon Mar  3 12:47:57 2025
NAMESPACE: jobset-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Thank you for installing jobset. The JobSet controller is now deployed in your cluster. To verify that the controller is running, you can run:
  kubectl get deployment -n jobset-system jobset-controller

```

Works for me.

Reply:
ok, forgot to set VERSION 🤦